### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by AlexWhitham**

### CODE REVIEW

File: ci-bootstrap/terraform/role.tf.tmpl [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                "iam:DeleteRolePolicy",
+                "iam:DeleteRole",
+                "iam:DeletePolicy",
+                "iam:CreateRole",
+                "iam:CreatePolicy",
+                "iam:AttachRolePolicy",
+                "apigateway:*"
+              ]
+    resources = [
+      "*"
```
Comment (necrophonic) [inline ci-bootstrap/terraform/role.tf.tmpl:95]: Can the target resources be locked down at all?

Comment (AlexWhitham) [reply ci-bootstrap/terraform/role.tf.tmpl:95]: I can try.

**Review by necrophonic — APPROVED** [review]
Can address looking at locking down the resources on the role as a separate PR

**Review by AlexWhitham — COMMENTED** [review]
Couldn't do it. Need to invest some time at some point to document and standardise policies/resources for different deployments.

**Review by bambrp — APPROVED** [review]
Added uat into readme and got rid of the now unused prefix var, and otherwise, LGTM, merging as soon as you sign that one commit missing your signature :)

**Pull request merged**